### PR TITLE
updated the all existing issue-templates

### LINKED ISSUE #46: Issue Template Updates for Figma link
### Current Behavior
The project's current issue templates are missing an open invitation link where new contributors can join the community's Figma team and view user interface designs and other UX projects.

### Desired Situation
Each template that has a reference to Figma in its resources section should an invite link added.

### Implementation 
```
 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

**Acceptance Tests**
All references to Figma include the "open invite" link.

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

### PR DESCRIPTION
**Pull request by sankhadip-roy**
**Description**
updated the all existing issue-templates with the contributing resources mentioned in Layer5.io's [issue_templates](https://github.com/layer5io/layer5/blob/master/.github/ISSUE_TEMPLATE/bug_report.md#contributor-resources-and-handbook)
This PR fixes #46

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -28,3 +28,13 @@ Steps to reproduce the behavior:
 3. Scroll down to '....'
 4. See error
 -->
+
+---
+<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]:
```suggestion
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -28,3 +28,13 @@ Steps to reproduce the behavior:
...
+
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 3]
```diff
@@ -10,7 +10,11 @@ assignees: ''
 **Desired State:**
 
 ---
-**Contributor Resources**
-- [Meshery documentation site](https://docs.meshery.io/)
-- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
-- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
+<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/documentation.md:13]:
```suggestion
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>
```

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 4]
```diff
@@ -10,7 +10,11 @@ assignees: ''
...
+
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/documentation.md:15]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/documentation.md (hunk 3 above)
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/documentation.md:13]:
```suggestion
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -21,4 +21,11 @@ assignees: ''
 <!-- [Optional] Any visual diagrams of the desired user interface. -->
 
 ---
-**[Contributor Guide](https://docs.meshery.io/project/contributing)**
+<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/feature_request.md:24]:
```suggestion
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -21,4 +21,11 @@ assignees: ''
...
+
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
+- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
+- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
+- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).
+
+<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/feature_request.md:31]:
```suggestion
Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).
```

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 7]
```diff
@@ -10,7 +10,11 @@ assignees: ''
...
+- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
+- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
+- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).
+
+<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/documentation.md:20]:
```suggestion
Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 8]
```diff
@@ -28,3 +28,13 @@ Steps to reproduce the behavior:
...
+- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
+- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
+- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).
+
+<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/bug_report.md:40]:
```suggestion
Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 9]
```diff
@@ -3,3 +3,13 @@ contact_links:
 - name: 🙋🏾🙋🏼‍Question
   url: https://github.com/layer5io/meshery/discussions/new
   about: Submit your question using GitHub Discussions.
+
+  ---
+<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/question.md:8]:
```suggestion
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 10]
```diff
@@ -3,3 +3,13 @@ contact_links:
...
+
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
+- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
+- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
+- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).
+
+<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/question.md:15]:
```suggestion
Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 11]
```diff
@@ -3,3 +3,13 @@ contact_links:
 - name: 🙋🏾🙋🏼‍Question
   url: https://github.com/layer5io/meshery/discussions/new
   about: Submit your question using GitHub Discussions.
+
+  ---
+<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>
+
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/question.md:10]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 12]
```diff
@@ -21,4 +21,11 @@ assignees: ''
 <!-- [Optional] Any visual diagrams of the desired user interface. -->
 
 ---
-**[Contributor Guide](https://docs.meshery.io/project/contributing)**
+<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>
+
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/feature_request.md:26]:
```suggestion

```

**Review by thisiskaransgit — COMMENTED** [review]
Thank you for your efforts
- Referencing the images is from another repository is not suggested, we can opt out of using it here
- Since this repository is not related to website, I removed it's mentions

Incorporate these changes and we're good to go

**Review by thisiskaransgit — APPROVED** [review]
@Sankhadip-Roy  Looks good to merge, thank you for your contribution :tada:

**Comment by Sankhadip-Roy** [comment]
> @Sankhadip-Roy Looks good to merge, thank you for your contribution tada

Thank you very much.

**Pull request merged**